test/routes_test.go

```diff
@@ -390,17 +390,26 @@ func TestRouteQueueSemantics(t *testing.T) {
 	routeSend("MSG foo RSID:1:1 2\r\nok\r\n")
 	// Queue group one.
 	routeSend("MSG foo QRSID:1:2 2\r\nok\r\n")
+	// Invlaid queue sid.
+	routeSend("MSG foo QRSID 2\r\nok\r\n")
```

Comment:
With the changes, those invalid SID, since they all start with `QRSID`, should not translate into a message to regular subscribers.